server/src/main/java/org/candlepin/guice/CandlepinFilterModule.java

```diff
@@ -57,6 +58,7 @@ protected void configureServlets() {
 
         filterRegex(regex).through(CandlepinScopeFilter.class);
         filterRegex(regex).through(CandlepinPersistFilter.class);
+        filterRegex(noStatusResource).through(CandlepinSuspendModeFilter.class);
```

Comment:
Since we want to throw unifrom Candlepin exception from the filter, I think better approach will be  to create JAX-RS PreMatching filter and not use ServletFilter here. Then we can use SuspendException that will be handled by ExceptionMapper